packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -365,14 +381,14 @@ describe('SnapController', () => {
     await service.terminateAllSnaps();
   });
 
-  it('should add a snap and use its JSON-RPC api with a NodeExecutionService', async () => {
+  it('adds a snap and uses its JSON-RPC api with a WebWorkerExecutionService', async () => {
```

Comment:
```suggestion
  it('adds a snap and uses its JSON-RPC api with a NodeExecutionService', async () => {
```

WebWorker is dead, long live NodeExecutionService